openstudiocore/src/project/ProjectDatabase.cpp

```diff
@@ -186,6 +186,7 @@ namespace detail {
     // make sure we are the last one using database connection
     OS_ASSERT(m_qSqlDatabase.use_count() == 1);
 
+    QString connectionName = m_qSqlDatabase->connectionName();
```

Comment:
@kbenne I was attempting to get rid of the error messages about closing QSqlDatabase connections still in use.  I did not solve this issue.